### PR TITLE
feat: custom twitter theme

### PR DESCRIPTION
**Pull request by nuanyang233**
closes Mask-339

### CODE REVIEW

**Review by Jack-Works — APPROVED** [review]

**Comment by nuanyang233** [comment]
The pr is testing

**Review by UncleBill — APPROVED** [review]

File: packages/mask/src/social-network-adaptor/twitter.com/customization/custom.ts [hunk 1]
```diff
@@ -60,11 +61,21 @@ export function useThemeTwitterVariant(baseTheme: Theme) {
                 contrastText: toRGB(primaryContrastColorRGB),
             }
             const themeName = isDark ? (isDarker ? 'darker' : 'dark') : 'light'
+
+            // Just for design
+            if (themeName === 'dark') {
+                theme.palette.background.paper = '#151D26'
```
Comment (zhouhanseng) [inline packages/mask/src/social-network-adaptor/twitter.com/customization/custom.ts:67]: Why not put `background.paper` in twitterColorSchema?

File: packages/mask/src/social-network-adaptor/twitter.com/customization/custom.ts [hunk 2]
```diff
@@ -60,11 +61,21 @@ export function useThemeTwitterVariant(baseTheme: Theme) {
...
             const colorSchema = twitterColorSchema[themeName]
             const colors = Object.keys(colorSchema) as Array<keyof typeof colorSchema>
             colors.forEach((color) => {
-                Object.assign(theme.palette[color], colorSchema[color])
+                if (typeof theme.palette[color] === 'object') {
+                    Object.assign(theme.palette[color], colorSchema[color])
+                }
             })
+            theme.palette.divider = colorSchema.divider
+            theme.palette.divider2 = colorSchema.divider2
```
Comment (zhouhanseng) [inline packages/mask/src/social-network-adaptor/twitter.com/customization/custom.ts:78]: `divider_darker` is more readable?

**Review by zhouhanseng — APPROVED** [review]

**Review by Jack-Works — APPROVED** [review]

File: packages/mask/src/social-network-adaptor/twitter.com/customization/custom.ts (hunk 2 above)
Comment (nuanyang233) [reply packages/mask/src/social-network-adaptor/twitter.com/customization/custom.ts:78]: Just to sync with figma

**Pull request merged**